docs/User Manual.rst

```diff
@@ -218,6 +218,9 @@ The possible types are:
     energy, the ratio between the local material cross section and the majorant cross
     section is larger than the cutoff, DT is used; otherwise ST is used.
 
+* handleVirtual (*optional*, default = 0): if set to 1 handles virtual as well as physical
+  collisions
+
```

Comment:
Now everything looks good, but I would fix the documentation a bit. I would move handleVirtual to the same level as '- cutoff', and add it under transportOperatorDT as well as transportOperatorHT. 
Also, in the description maybe it's more precise to say: 'if set to 1, delta tracking virtual collisions are tallied with a collisionClerk as well as physical collisions'

Reply:
Cheers, fixed now.